storage-node/src/cache/replacer/mod.rs

```diff
@@ -37,6 +37,10 @@ pub trait DataStoreReplacer<K: ReplacerKey, V: ReplacerValue>: Send + Sync {
     /// Returns `Some`: insertion successful with a list of keys that are evicted from the cache.
     fn put(&mut self, key: K, value: V) -> Option<Vec<K>>;
 
+    fn pin(&mut self, key: &K, count: usize) -> bool;
+
+    fn unpin(&mut self, key: &K) -> bool;
```

Comment:
Why do we pin by `count` and always unpin by one?

Reply:
since it is more convenient to use for MemDiskCache